### ISSUE TITLE
Absence of ''Contacts" attribute in statistics menu 

### ISSUE THREAD

**Issue by Iheanacho2027** [hzi-braunschweig/SORMAS-Project#1281]
There is no "contacts" attribute in statistics menu filtering options. There is a need for this because in future, data management analyst may need to carry out statistical analysis as regards the number of followed-up and unfollowed contacts available on the SORMAS platform. There should be also be sub-attributes for followed-up, unfollowed and lost to follow-up contacts
![image c3](https://user-images.githubusercontent.com/50739609/63804158-71f33480-c90e-11e9-82e7-e8a77aefd3c1.JPG)

**Comment by MateStrysewskeSym** [comment]
@Iheanacho2027 That's because the statistics section right now exclusively supports cases, none of the other entities in SORMAS. So this is more a matter of extending the statistics to also include information about contacts which I absolutely see the reasoning for. I've renamed the issue accordingly.

**Comment by Iheanacho2027** [comment]
Also in addition to this, can we also have information on sample results in the statistics menu

**Comment by foyiri** [comment]
yes the lab forks should be able to carry out sample analysis on the platform like how many sample were received how many were test  etc

**Comment by faatihi** [comment]
users should be able to extract statistics of contacts **per region/district** for a particular disease

**Comment by faatihi** [comment]
An example of statistics that might be needed from contacts

![image](https://user-images.githubusercontent.com/1176692/77935723-5b94c900-72a1-11ea-9921-59b80e719666.png)

**Comment by faatihi** [comment]
I've come up with two UI design proposals

1. Tabbed statistics

![tabbed](https://user-images.githubusercontent.com/1176692/78137476-eb0eb900-7414-11ea-9d8f-852c380af4ba.png)

2. Using grouped-buttons

![grouped-buttons](https://user-images.githubusercontent.com/1176692/78137519-fc57c580-7414-11ea-8e9f-f4a8cbb5e201.png)

I would like second/third opinions

**Comment by MartinWahnschaffeSymeda** [comment]
Clear vote on the first option (tabbed)

**Comment by MateStrysewskeSym** [comment]
Seconding this.

**Comment by Iheanacho2027** [comment]
I prefer the second because it looks more detailed

**Comment by foyiri** [comment]
I think the second looks more better

**Comment by MateStrysewskeSym** [comment]
@Iheanacho2027 They're both exactly the same, the only difference is that in option A), you choose whether to get case or contact statistics via a tab menu while in option B), you make the same choice in a switch field.

**Comment by Iheanacho2027** [comment]
Yeah i know. I thought he was asking which U.I design is more preferred

**Comment by faatihi** [comment]
Thanks for your input.
I've gone for tabbed. It's easier to reason about ...from developer's point.
